src/material/menu/menu.ts

```diff
@@ -237,6 +239,8 @@ export class _MatMenuBase implements AfterContentInit, MatMenuPanel<MatMenuItem>
    */
   @Output() close = this.closed;
 
+  readonly panelId = `mat-menu-panel-${menuPanelUid++}`;
```

Comment:
Should this be underscored?

Reply:
I think if someone makes there own custom menu we want them to provide this property too, so I think its fair to leave it without the underscore.